src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,52 @@
+package duke.command;
+
+import duke.component.TaskList;
+import duke.component.Storage;
+import duke.component.Ui;
+import duke.component.GuiResponse;
+
+import duke.task.Task;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Command Class for adding tasks.
+ */
+public class AddCommand extends Command {
+
+    private Task newTask;
+
+    /**
+     * Constructor for AddCommand Object.
```

Comment:
I think this is something minor, but it would be good to follow some of the intermediate rules for coding standards for JavaDoc comments. I think by having an empty line between the description and parameter section, it can help make our long blocks of comments easier to read.